### PR TITLE
[VDC-5203] Feature updated from V5 - Setting of licenseType on Snapshots at time of creation

### PR DESCRIPTION
**Pull request by mdomentii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1437,8 +1437,9 @@ Creates a snapshot of a volume within the data center. You can use a snapshot to
 |---|---|---|---|
 | dataCenterId | **yes** | string | The ID of the datacenter. |
 | volumeId | **yes** | string | The ID of the volume. |
-| name |  no | string | The name of the snapshot. |
+| name |  **yes** | string | The name of the snapshot. |
 | description | no | string | The description of the snapshot. |
+| licenceType | no | string | Set to one of the values: [WINDOWS, WINDOWS2016, LINUX, OTHER, UNKNOWN] |
```
Comment (cristianSaru) [inline README.md]: should we have "UNRECOGNIZED" type here or it is used only for technical purpose ?

File: src/test/java/com/ionosenterprise/rest/test/SnapshotTest.java [hunk 2]
```diff
@@ -84,7 +80,7 @@ public static void t1_createSnapshot() throws RestClientException, IOException,
 
         Snapshot.Properties properties = SnapshotResource.getSnapshot().getProperties();
         Snapshot snapshot = ionosEnterpriseApi.getSnapshot().createSnapshot(dataCenterId, volumeId,
-                properties.getName(), properties.getDescription());
+                properties.getName(), properties.getDescription(), properties.getLicenceType().name());
         assertNotNull(snapshot);
         assertEquals(snapshot.getProperties().getName(), properties.getName());
         assertEquals(snapshot.getProperties().getDescription(), properties.getDescription());
```
Comment (cristianSaru) [inline src/test/java/com/ionosenterprise/rest/test/SnapshotTest.java]: i think we also need an assert here for the licence type , correct me if i am wrong

File: README.md (hunk 1 above)
Comment (mdomentii) [reply README.md]: The only accepted values for the licenceType field accordingly Public API v5 documentation are: WINDOWS, WINDOWS2016, LINUX, OTHER, UNKNOWN. There is no UNRECOGNIZED value in the documentation.

File: src/test/java/com/ionosenterprise/rest/test/SnapshotTest.java (hunk 2 above)
Comment (mdomentii) [reply src/test/java/com/ionosenterprise/rest/test/SnapshotTest.java]: You are right. Fixed.

**Pull request merged**